server/schedule/operator.go

```diff
@@ -36,6 +36,7 @@ type OperatorStep interface {
 	fmt.Stringer
 	IsFinish(region *core.RegionInfo) bool
 	Influence(opInfluence OpInfluence, region *core.RegionInfo)
+	InvolvedStores(region *core.RegionInfo) []uint64
```

Comment:
How about `AffectStores`?

Reply:
It's kind weird to have a `region` parameter here. I think you can add a `OriginLeader` field to operators that need original leader id to return affect stores.